### PR TITLE
Revise documentation for inter-frame messages.

### PR DESCRIPTION
**Pull request by robertknight**
Revise the documentation in order to:

 - Reduce uninformative noise ("the X frame informs the Y frame to..."). The source and destination frame type are given by the type name.

 - Re-phrase the documentation to read more like either a function description or an event description, depending on whether the message is a specific command/request or a more general event.

 - Use the plural "guests" to make it clear that there can be more than one guest frame.

 - Clarify when events specific to third-party authorities are used.

### CODE REVIEW

File: src/types/port-rpc-events.d.ts [hunk 1]
```diff
@@ -26,17 +26,15 @@ export type GuestToHostEvent =
   | 'highlightsVisibleChanged';
 
 /**
- * Events that the guest sends to the sidebar
+ * Events that guests send to the sidebar.
  */
 export type GuestToSidebarEvent =
```
Comment (acelaya) [inline src/types/port-rpc-events.d.ts:31]: We probably need to revisit the values here, as these are both events and commands, but they are inconsistently spelled as commands or events.

* `closeSidebar`, `openSidebar` -> These are in fact commands where guests request an action to be performed.
* `createAnnotation` -> This is an event, so I guess it should be renamed as `annotationCreated`?
* `documentInfoChanged` -> This is in fact an event, as the name suggests.

File: src/types/port-rpc-events.d.ts [hunk 2]
```diff
@@ -26,17 +26,15 @@ export type GuestToHostEvent =
   | 'highlightsVisibleChanged';
 
 /**
- * Events that the guest sends to the sidebar
+ * Events that guests send to the sidebar.
```
Comment (acelaya) [inline src/types/port-rpc-events.d.ts:29]:
```suggestion
 * Events and commands that guests send to the sidebar.
```

File: src/types/port-rpc-events.d.ts [hunk 3]
```diff
@@ -45,37 +43,35 @@ export type GuestToSidebarEvent =
...
   | 'toggleAnnotationSelection';
 
 /**
- * Events that the host sends to the guest
+ * Events that the host sends to guests.
  */
 export type HostToGuestEvent =
-  /** The host requests a guest to create an annotation. */
+  /**
+   * The host requests that the guest frame create a new annotation.
```
Comment (acelaya) [inline src/types/port-rpc-events.d.ts:66]:
```suggestion
   * The host requests that the guest frame creates a new annotation.
```

**Review by acelaya — APPROVED** [review]
It looks good in general.

I just added a suggestion on eventually reviewing the message names, to use the imperative form for commands, and past tense for events. This is currently inconsistent.

**Comment by robertknight** [comment]
Thanks for the review. I'm going to merge this as-is but I agree that some of the message names could do with an update.

The distinction between an "event" and a "command" is fuzzy in some cases, as certain messages could be framed either way.

**Comment by acelaya** [comment]
> The distinction between an "event" and a "command" is fuzzy in some cases, as certain messages could be framed either way.

Yeah, this is also true.

**Pull request merged**